src/source/reference/get-metrics-ads.rst

```diff
@@ -54,8 +73,37 @@ fields          list of strings      no                 Comma-separated list of
 ==============  ================  ==================  ======================================================================================================================================================================================================================================================================================================
 
 
-Report Columns
-~~~~~~~~~~~~~~
+Report Columns V2
+~~~~~~~~~~~~~~~~~
+
+Both the excel and the csv formats contain the following columns by default:
+
+====================   =================   ===============================================================
+Name                   Field               Description
+====================   =================   ===============================================================
+Date (Aggregated)      ``date``            Date of the report row, grouped daily, weekly, monthly, or yearly. For daily and weekly aggregation the format is ``YYYY-MM-DD``, for monthly aggregation - ``YYYY-MM``, and for yearly - ``YYYY``. All dates are in tenant timezone.
+Ad ID                  ``adID``            ID of the ad
+L1 Category            ``L1Category``      Level 1 category name
+L2 Category            ``L2Category``      Level 2 category name
+L3 Category            ``L3Category``      Level 3 category name, if applicable
+Title                  ``title``           Title of the ad
+Start Date             ``startDate``       Creation date of the ad
+End Date               ``endDate``         If the ad is deleted, deletion date of the ad
+Bid (Micros)           ``bidMicros``       Bid of the ad for which performance metrics are calculated, in Micros
+Total Spent (Micros)   ``spentMicros``     Total amount spent for this ad, in Micros
+Clicks                 ``clicks``          Number of clicks that the ad received
+Impressions            ``impressions``     Number of impressions that the ad received
+CTR                    ``viewCTR``         Click-through rate in %
+URL clicks             ``websiteClicks``   Number of URL clicks that the ad received
+Emails                 ``emails``          Number of email events that the ad received
+Engagement CTR         ``engagementCTR``   Engagement conversion rate in %. Calculation: ``(URL clicks + Emails) / Clicks``
+Region                 ``region``          Region name, of the ad
+Vendor ID              ``vendorID``        Vendor ID of the ad
+====================   =================   ===============================================================
+
+
+Report Columns V1
```

Comment:
Everything V2 related should be on top, followed by V1 on the bottom. If there is an overlap in the general descriptin - either keep it in both places, or better yet - reduce the V1 content